client/src/components/InventoryTable.js

```diff
@@ -48,7 +44,6 @@ class InventoryTable extends React.Component {
     return (
```

Comment:
Add a new method to InventoryTable like so:
```
class InventoryTable extends React.Component {
  handleAddItem(item, e) {
    this.props.onAddItem(item)
  }
  ...
}
```

With this new method plus the change to `onClick` your button should now work.